Server/Python/src/dbs/business/DBSBlock.py

```diff
@@ -8,6 +8,16 @@
 from dbs.utils.dbsExceptionHandler import dbsExceptionHandler
 import re
 
+
+def convertByteStr(unicodeStr):
+    """
+    Utilitarian function which converts an unicode string to
+    an 8-bit string.
+    """
+    if isinstance(unicodeStr, basestring):
```

Comment:
`basestring` is not supported in python3, instead better use `str`

Reply:
Yes, but at least we know this is a case that will hit us again when we move to python3. So it's easier to identify when we start the real migration.